src/class_singleCleaner.py

```diff
@@ -57,7 +57,7 @@ def run(self):  # pylint: disable=too-many-branches
                 float(BMConfigParser().get(
                     'bitmessagesettings', 'stopresendingafterxmonths'))
                 * (60 * 60 * 24 * 365) / 12)
-        except:
+        except:  # noqa:E722
```

Comment:
add a method `BMConfigParser.safeGetFloat`, also add a default value into `BMConfigDefaults` and then there is no need for error handler because it will handle missing as well as invalid values.

Reply:
Updated.